lua/wikis/commons/Infobox/Person.lua

```diff
@@ -471,12 +560,14 @@ end
 ---@param status PlayerStatus
 ---@return string[]
 function Person:getCategories(args, birthDisplay, personType, status)
+	local categories = {}
 	if not self:shouldStoreData(args) then
-		return {}
+		return categories
 	end
-
 	local team = args.teamlink or args.team
-	local categories = Array.append(self.age.categories,
+
+	categories = Array.extend(categories, self.age.categories)
+	categories = Array.extend(categories,
```

Comment:
Not sure if all wikis want it, but at least AoE would like to have the category for every role added, not just the personType